webapp/client/src/lib/propTypes.js

```diff
@@ -1,15 +1,33 @@
 import PropTypes from "prop-types";
+import FieldLabelScaffolding from "../components/FieldLabelScaffolding";
+
+// Used if the field is handed to a more abstract component.
+// e.g. MerkzeichenPersonAPage -> MerkzeichenPage
+const abstractedFieldPropType = PropTypes.exact({
+  name: PropTypes.string.isRequired,
+  label: FieldLabelScaffolding.propTypes.label,
+});
```

Comment:
At first I was not so sure about this but I think I like it. However, could maybe put it at the end of this file. If reading from top to bottom you would at first see the usage of that and I think this would help with understanding why it exists

Reply:
Then the constant is used before it is defined. Or what did you mean?

Reply:
Nevermind then.